extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/security/PathMatchingHttpSecurityPolicyTest.java

```diff
@@ -190,25 +185,6 @@ public void testStaticResource(String path) {
         assurePathAuthenticated(path);
     }
 
-    @ParameterizedTest
```

Comment:
Why are they moved?

Reply:
So, here is the story on why they moved. The tests is in vertx-http, but (test only) need dependencies on openapi and health, both of which depend on vertx-http. The Windows build failed with this error, however the Linux build did not. I am not sure if the Linux build has some cache from a prev build (no-test) on CI, and Windows does not or why this only happens on Windows, but I could recreate it locally on a VM. It also seem that the failure in Windows is triggered by the new dependency added to the vertx-deploy module. I have no idea why. So to "fix" (sidestep) this, I moved the openapi test to openapi and the heath to health. This solved the issue, but i still can not say why this happened, only on Windows and triggered by the dependency.

Reply:
@cescoffier are you ok to merge with the explanation above ?